README.md

```diff
@@ -25,94 +25,138 @@
 <br />
 <div align="center">
   <a href="https://github.com/applied-exploration/modular-pipelines">
-    <!-- <img src="images/logo.png" alt="Logo" width="80" height="80"> -->
+    <img src="docs/resources/logo.png" alt="Logo" width="160" >
```

Comment:
do we want to mix html with markdown?

Reply:
anyway, let's do it!

Reply:
There are some stuff that we cannot do with markdown only, but it's more convenient to use markdown when there is just text.